client/blocks/upsell-nudge/index.jsx

```diff
@@ -60,7 +60,15 @@ export const UpsellNudge = ( {
 		href = addQueryArgs( { feature, plan }, `/plans/${ site.slug }` );
 	}
 
-	return <Banner { ...props } showIcon={ showIcon } className={ classes } href={ href } />;
+	return (
+		<Banner
+			{ ...props }
+			showIcon={ showIcon }
+			disableHref={ ! canManageSite }
```

Comment:
I think we should leave it up to the component instance to set `disableHref` rather than doing so globally. I would move this prop into the component in `client/components/seo/preview-upgrade-nudge/index.jsx`.

This is the first upsell I've seen like this, though. I don't think it's common for us to advertise to a group that can't act on that advertisement beyond contacting the site administrator. I could very well be wrong, though!

Let me know what you think, or if there's another reason for setting this globally. :)

Reply:
I also don't think it's very common; there were a few instances added in #34597, but they mainly refer to `EmptyContent`, so I think that it's fine to move it to individual nudges! I've made that change now. :) 

That being said, I don't think there ever should be a case though where someone who can't access Plans is able to click an upgrade nudge.

Reply:
> That being said, I don't think there ever should be a case though where someone who can't access Plans is able to click an upgrade nudge.

True true. I don't know off the top of my head if `disableHref` applies when there's a call-to-action button instead of a linked banner, or how it interacts with `forceHref`. There are many, uh, layers to the logic in these things 🙃. But if it's declared at the top level it's more visible and easier to parse, I would think.

Reply:
Also true! :)